RAQUASI88/src/keyboard.c

```diff
@@ -1866,6 +1866,7 @@ int stateload_keyboard( void )
 static const T_SYMBOL_TABLE key88sym_list[] =
 {
     { "KEY88_INVALID"       ,   KEY88_INVALID       },
+    { "KEY88_DASH"      ,   KEY88_DASH              },
```

Comment:
minor: alignment

Reply:
See the lines below it, alignment is all over the place in this table (and in many parts of the project). I matched the alignment of other short definitions.

Reply:
ok. lines below weren't immediately visible in the review. seems like a tab vs spaces thing, but everything is spaces now.